config.sample

```diff
@@ -11,11 +11,12 @@ OWNER=
 DATASET_OWNER=
 # the master developer key (to rebuild images)
 THINGPEDIA_DEVELOPER_KEY=
-# the Genie/decanlp task to train: either `almond` or `contextual_almond`
+# the Genie/decanlp task to train: either `almond` or `contextual_almond` or `almond_dialogue_nlu`
 TRAIN_TASK_NAME=almond
+EVAL_TASK_NAME=almond
```

Comment:
Is EVAL_TASK_NAME used at all?

Reply:
I'll have to remove a few unused arguments.